projectile.el

```diff
@@ -4527,11 +4527,16 @@ See `projectile--cleanup-known-projects'."
     (message "No projects needed to be removed.")))
 
 ;;;###autoload
-(defun projectile-clear-known-projects ()
-  "Clear both `projectile-known-projects' and `projectile-known-projects-file'."
-  (interactive)
+(defun projectile-clear-known-projects (rediscover)
```

Comment:
You need to make this `&optional` to avoid breaking existing clients. That's also why the tests are failing.